libvirt/README.md

```diff
@@ -0,0 +1,53 @@
+# Machine Stats for Virtual environments (Virt-Stats)
+
+A simple and effective way to gather guest VM statistics (hostname, IP addresses) from a [libvirt](https://libvirt.org/)-based environment as a first layer of a [Tidal Migrations discovery process](https://guides.tidalmg.com/).
+
+## Prerequisutes
+
+* [libvirt](https://libvirt.org/) >=3.0.0 installed on both operator machine (i.e where you run Virt-Stats) and on remote machine (i.e where you run your virtual environmnent and guest VMs)
+* Every guest VM must have [QEMU guest agent](https://wiki.qemu.org/Features/GuestAgent) installed and running.
```

Comment:
I wondering if we should include some commands to check whether QEMU is installed or not.  WDYT?

Reply:
No longer relevant, since QEMU guest agent installation is optional now